cypress/integration/pages/organizations.spec.js

```diff
@@ -0,0 +1,262 @@
+    cy.intercept(`${Cypress.env('REACT_APP_API_URL')}/api/organizations/`).as('getOrganizations');
+    cy.visit('/organizations/all');
+    cy.wait('@getOrganizations');
+    cy.get('[data-cy=index-contributors-checkbox] input:checkbox').uncheck();
+    cy.contains(grandparentOrg).should('have.length', 1);
+    cy.get('[href*=codeforall]').should('have.length', 1);
+    cy.get('[data-cy=code-for-all-chevron]').click();
+    cy.get('[data-cy=affiliated-organizations]').within(() => {
+      cy.get('[data-cy=thumbnail-dropdown]').should('have.length', parentOrgCount);
+      cy.get('[data-cy=thumbnail-dropdown]')
+        .contains(parentOrg)
+        .parentsUntil('[data-cy=thumbnail-dropdown]')
+        .within(() => {
+          cy.get('[data-cy=dropdown-chevron]').click();
+        });
+      cy.contains(affiliatedOrg);
+    });
+  });
+
+  it('should find affiliatedOrg via search', () => {
+    cy.intercept(`${Cypress.env('REACT_APP_API_URL')}/api/organizations/`).as('getOrganizations');
+    cy.visit('/organizations/affiliated');
+    cy.wait('@getOrganizations');
+    cy.get('[data-cy=organization-search]').type(affiliatedOrg);
+    cy.get('[class*=makeStyles-gpGrid]').click();
+    cy.get('[data-cy=affiliated-organizations]').within(() => {
+      cy.get('[data-cy=thumbnail-dropdown]').should('have.length', 1);
+      cy.get('[data-cy=contributor-thumbnail-text]').contains(affiliatedOrg);
+    });
+    cy.get('[class*=endAdornment]').click().within(() => {
+      cy.get('button').click();
+    });
+    cy.get('[data-cy=affiliated-organizations]').within(() => {
+      cy.get('[data-cy=thumbnail-dropdown]').should('have.length', parentOrgCount);
+    });
+  });
+
+  it('should find affiliatedOrg via partial search', () => {
+    cy.intercept(`${Cypress.env('REACT_APP_API_URL')}/api/organizations/`).as(
+      'getOrganizations'
+    );
+    cy.visit('/organizations/affiliated');
+    cy.wait('@getOrganizations');
+    cy.get('[data-cy=organization-search]').type(affiliatedOrgPartial);
+    cy.get('[data-cy=organization-search-list]').should('have.length', 1);
+    cy.get('[data-cy=organization-search-list]').click();
+    cy.get('[class*=makeStyles-gpGrid]').click();
+    cy.get('[data-cy=affiliated-organizations]').within(() => {
+      cy.get('[data-cy=thumbnail-dropdown]').should('have.length', 1);
+      cy.get('[data-cy=contributor-thumbnail-text]').contains(affiliatedOrg);
+    });
+  });
+
+  it('should visit organizations page from home page link', () => {
+    cy.intercept(`${Cypress.env('REACT_APP_API_URL')}/api/organizations/`).as(
+      'getOrganizations'
+    );
+    cy.visit('/home');
+    cy.contains('View contributors').click();
+    cy.wait('@getOrganizations');
+    cy.get('[class*=pageContainer]').should(
+      'contain',
+      'Want to add your organization'
+    );
+  });
+});
+
+describe('Contributors Page (using fixture)', () => {
```

Comment:
Getting eslint max lines per function error here.  Perhaps the eslint comment should be moved from the top of the file to here?

![image](https://user-images.githubusercontent.com/1535252/129494109-fb385b8d-5c91-41b4-b33d-38a9431fff6d.png)

Reply:
Yup!